### PR TITLE
Code of conduct agreement

### LINKED ISSUE #218: Add checkbox to new session form for CoC agreement
Each presenter needs to agree to this events CoC.

https://minnestar.org/coc

How do we enforce this agreement for multiple presenters? Sign in/registration? Or just on new session creation?

### PR DESCRIPTION
**Pull request by aackerman**
Fixes #218

### CODE REVIEW

File: src/app/models/participant.rb [hunk 1]
```diff
@@ -43,6 +43,8 @@ def deliver_password_reset_instructions!
   end
 
   def signed_code_of_conduct_for_current_event?
+    return false if Event.current_event.nil?
```
Comment (jcoyne) [inline src/app/models/participant.rb]: how about `unless Event.current_event` ?

**Comment by jennapederson** [comment]
Hey @aackerman - What's the scoop on some of these Travis failures? Wondering if the setup is still in progress or if we're good to merge some of these even with it failing.

**Comment by aackerman** [comment]
@jennapederson at some point the default build image on travis changed from ubuntu trusty to ubuntu xenial and the CI setup stopped working, looks like about 5 months ago. I've added configuration to run on ubuntu trusty.

**Comment by jennapederson** [comment]
@aackerman I want to make sure I understand how this will work in real life... a person creates a session and goes to add another presenter to the session. they won't be able to until that person has agreed to the coc, right? and each coc agreement is for an event, right?

what happens if we push this to production after some sessions/presenters have already been created/added? they'll be ok, but any new sessions/presenters added will have to agree?

I think that's ok, but want to be able to message Maria and others who get to field questions in case something comes up!

Thank you for working on this!

**Comment by aackerman** [comment]
@jennapederson it's been a few months but yes that's how I expect it to work, new sessions and new participants will have to agree.

**Comment by jennapederson** [comment]
@aackerman Same for this -- can you fix conflicts and then we'll get it merged? Thanks!

**Pull request merged**